trulens_eval/trulens_eval/feedback/benchmark_frameworks/meta_evaluation_examples.ipynb

```diff
@@ -0,0 +1,305 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Meta Evaluation - evaluating your LLM-as-judge with TruLens\n",
+    "\n",
+    "Meta evaluation is the process of evaluating evaluation methods themselves. Here we are measuring and benchmarking the performance of LLM-based evaluators (aka LLM-as-judge), where the main focus of performance is human alignment. In other words, how closely aligned the generated scores are with human evaluation processes.\n"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "###\n",
+    "In TruLens, we implement this as a special case of GroundTruth evaluation, since we canonically regard human preferences as the groundtruth in most LLM tasks. \n",
+    "\n",
+    "For experiment tracking, we provide a suite automatic metric computation via Aggregator, "
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Import relevance feedback function\n",
+    "from trulens_eval.feedback import GroundTruthAgreement, GroundTruthAggregator\n",
+    "from trulens_eval import Tru\n",
+    "import numpy as np\n",
+    "\n",
+    "tru = Tru()\n",
+    "tru.reset_database()\n",
+    "\n",
+    "golden_set = [\n",
+    "    {\n",
+    "        \"query\": \"who are the Apple's competitors?\",\n",
+    "        \"response\": \"Apple competitors include Samsung, Google, and Microsoft.\",\n",
+    "        \"expected_score\": 1.0,  # groundtruth score annotated by human\n",
+    "    },\n",
+    "    {\n",
+    "        \"query\": \"what is the capital of France?\",\n",
+    "        \"response\": \"Paris is the capital of France.\",\n",
+    "        \"expected_score\": 1.0,\n",
+    "    },\n",
+    "    {\n",
+    "        \"query\": \"what is the capital of Spain?\",\n",
+    "        \"response\": \"I love going to Spain.\",\n",
+    "        \"expected_score\": 0,\n",
+    "    },\n",
+    "]\n",
+    "# Create a Feedback object using the numeric_difference method of the ground_truth object\n",
+    "ground_truth = GroundTruthAgreement(golden_set)"
```

Comment:
This is never used. What's the point of `GroundTruthAgreement`.

Reply:
removed